### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by Xleby4ek** [JannisX11/blockbench#2387]
### What are you trying to do, and what do you expect to happen?

I'm trying to run blockbench and expect it to run.

### What happens instead?

When running Blockbench on a linux system, the program crashes out 

### Model format in which the issue occurs

-

### Blockbench variant

Program

### Blockbench version

4.10.3 Deb

### Operating System

Linux Mint 21.3 x86_64

### Installed Blockbench plugins

None

**Comment by Xleby4ek** [comment]
This also happens in the AppImage version

**Comment by Xleby4ek** [comment]
I also tried deleting all files that had anything to do with blockbench, but it didn't work

**Comment by JannisX11** [comment]
I am unable to reproduce this issue.

**Comment by nab138** [comment]
This issue is happening for me too, also on Linux mint 21.3. When running from the terminal:
```❯ blockbench
Checking for beta autoupdate feature for deb/rpm distributions
Found package-type: deb
[1]    171627 segmentation fault (core dumped)  blockbench
```

**Comment by Xleby4ek** [comment]
@JannisX11 Are you going to fix this bug?
Apparently it happens ONLY in Linux Mint 21.3 since you have not been able to replicate this bug. 
Here is more information
When running blockbench through the console, the following message is displayed 
```
Checking for beta autoupdate feature for deb/rpm distributions
Found package-type: deb
Segmentation error (memory image dumped to disc)
```
If run via sudo blockbench
```
[0710/064017.701766:FATAL:electron_main_delegate.cc(295)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.
Trace/stop trap
```
If run via sudo blockbench --no-sandbox
```
Checking for beta autoupdate feature for deb/rpm distributions
Found package-type: deb
[6682:0710/064254.061611:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[6682:0710/064254.062028:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[6682:0710/064254.062067:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[6682:0710/064254.062098:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
Authorization required, but no authorization protocol specified
Authorization required, but no authorization protocol specified
Authorization required, but no authorization protocol specified
Authorization required, but no authorization protocol specified
Authorization required, but no authorization protocol specified
[6708:0710/064254.198208:ERROR:angle_platform_impl.cc(44)] Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Could not open the default X display.
ERR: Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Could not open the default X display.
[6708:0710/064254.198351:ERROR:gl_display.cc(515)] EGL Driver message (Critical) eglInitialize: Could not open the default X display.
[6708:0710/064254.198438:ERROR:gl_display.cc(786)] eglInitialize OpenGL failed with error EGL_NOT_INITIALIZED, trying next display type
Authorization required, but no authorization protocol specified
[6708:0710/064254.198819:ERROR:angle_platform_impl.cc(44)] Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Could not open the default X display.
ERR: Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Could not open the default X display.
[6708:0710/064254.198912:ERROR:gl_display.cc(515)] EGL Driver message (Critical) eglInitialize: Could not open the default X display.
[6708:0710/064254.198981:ERROR:gl_display.cc(786)] eglInitialize OpenGLES failed with error EGL_NOT_INITIALIZED
[6708:0710/064254.199058:ERROR:gl_display.cc(820)] Initialization of all EGL display types failed.
[6708:0710/064254.199128:ERROR:gl_ozone_egl.cc(26)] GLDisplayEGL::Initialize failed.
Authorization required, but no authorization protocol specified
[6708:0710/064254.263313:ERROR:angle_platform_impl.cc(44)] Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Could not open the default X display.
ERR: Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Could not open the default X display.
[6708:0710/064254.263449:ERROR:gl_display.cc(515)] EGL Driver message (Critical) eglInitialize: Could not open the default X display.
[6708:0710/064254.263522:ERROR:gl_display.cc(786)] eglInitialize OpenGL failed with error EGL_NOT_INITIALIZED, trying next display type
Authorization required, but no authorization protocol specified
[6708:0710/064254.263896:ERROR:angle_platform_impl.cc(44)] Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Could not open the default X display.
ERR: Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Could not open the default X display.
[6708:0710/064254.263991:ERROR:gl_display.cc(515)] EGL Driver message (Critical) eglInitialize: Could not open the default X display.
[6708:0710/064254.264064:ERROR:gl_display.cc(786)] eglInitialize OpenGLES failed with error EGL_NOT_INITIALIZED
[6708:0710/064254.264127:ERROR:gl_display.cc(820)] Initialization of all EGL display types failed.
[6708:0710/064254.264195:ERROR:gl_ozone_egl.cc(26)] GLDisplayEGL::Initialize failed.
[6708:0710/064254.265866:ERROR:viz_main_impl.cc(198)] Exiting GPU process due to errors during initialization
[6682:0710/064255.114178:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[6682:0710/064255.203679:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
Authorization required, but no authorization protocol specified
Authorization required, but no authorization protocol specified
Authorization required, but no authorization protocol specified
Authorization required, but no authorization protocol specified
Authorization required, but no authorization protocol specified
[6731:0100/000000.419907:ERROR:angle_platform_impl.cc(44)] Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Could not open the default X display.
ERR: Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Could not open the default X display.
[6731:0100/000000.648836:ERROR:gl_display.cc(515)] EGL Driver message (Critical) eglInitialize: Could not open the default X display.
[6731:0100/000000.648876:ERROR:gl_display.cc(786)] eglInitialize OpenGL failed with error EGL_NOT_INITIALIZED, trying next display type
Segmentation error
```

**Comment by RepoRubbernecker** [comment]
I'm also on Linux Mint 21.3 (specifically Cinnamon), when starting Blockbench it opens a blank window before closing out.

**Comment by TimeBender25** [comment]
Bump, been having this issue since Blockbench 4.10.0 and newer. I don't use the app professionally, so I've been content to downgrade to 4.9.4 (which still works fine), but it's a shame to not be able to access the newer tools and plugins.

I'm running Linux Mint 21 (Vanessa), with Cinnamon 5.4.12. I get the exact same error as @Xleby4ek when trying to run from Terminal. It'll open to the empty grey screen for a moment before crashing out with the message, "Segmentation Fault".

**Comment by JannisX11** [comment]
This should be fixed by upgrading electron to `30.0.5` or newer (https://github.com/electron/electron/issues/41839). Upgrading to newer electron versions has been held up by (https://github.com/JannisX11/blockbench/issues/2364) so far, which is another electron issue, so I need to make sure that one is fixed before upgrading.
